src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -40,26 +37,23 @@ export const NormalAnnouncementBubble: React.FC<Props> = ({
   const replyButton = formReplyButton(onReply, repliesDisabledMsg);
 
   return (
-    <FlexBox grow column>
```

Comment:
The only change was dropping this outer container and the error on line 62, which was moved to MessageMetadata.